ports/stm32/Makefile

```diff
@@ -471,6 +471,12 @@ endif
 ifeq ($(MICROPY_SSL_MBEDTLS),1)
 CFLAGS_MOD += -DMBEDTLS_CONFIG_FILE='"mbedtls/mbedtls_config.h"'
 SRC_MOD += mbedtls/mbedtls_port.c
+ifeq ($(MICROPY_SSL_MBEDTLS_ERRSTR),1)
```

Comment:
Honestly, I think this should just be enabled unconditionally.  Boards that have SSL have a large amount of flash (because mbedtls is already large) and IMO it's worth having these error messages if you have SSL.

And anyway it's already configurable via `MBEDTLS_ERROR_C` which a board could potentially override if it wanted its own configuration for mbedtls.

So I'd suggest to just remove this config, put `#define MBEDTLS_ERROR_C` in `ports/stm32/mbedtls/mbedtls_config.h`, and unconditionally replace `error.c` with `mp_mbedtls_errors.c` (and that last part can be done unconditionally anyway, because if `MBEDTLS_ERROR_C` is disabled it just won't be used).